src/containers/CardDetail/CardDetail.jsx

```diff
@@ -6,10 +6,12 @@ import { deleteCard, editCard } from '../../actions';
 class CardDetail extends Component {
   componentWillMount() {
     document.addEventListener('mousedown', this.handleClick, false);
+    document.querySelector('.columnContainer').classList.add('blur');
   };
 
   componentWillUnmount() {
     document.removeEventListener('mousedown', this.handleClick, false);
+    document.querySelector('.columnContainer').classList.remove('blur');
```

Comment:
commented
Instead of using document, try to directly add the class in the render function using className and the ternary operator